lfx-mentorship/2023/01-Mar-May/project_ideas.md

```diff
@@ -15,3 +15,14 @@
 
 ## Proposed Project ideas
 
+### Cluster API Provider GCP (CAPG)
+
+#### Add telemetry and profiling support
```

Comment:
Could we add this under a 'Kubernetes' section in case any other sigs make a proposal?
```suggestion
### Kubernetes

#### Cluster API Provider GCP (CAPG)

##### Add telemetry and profiling support
```

Reply:
Good idea 👍